src/basis_factorization/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ target_include_directories(${MARABOU_TEST_LIB} PUBLIC "${CMAKE_CURRENT_SOURCE_DI
  
 set (BASIS_FACTORIZATION_TESTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/tests")
 macro(basis_factorization_add_unit_test name)
-    marabou_add_test(${BASIS_FACTORIZATION_TESTS_DIR}/Test_${name} basis_factorization TRUE "unit")
+    marabou_add_test(${BASIS_FACTORIZATION_TESTS_DIR}/Test_${name} basis_factorization TRUE TRUE "unit")
```

Comment:
This "TRUE TRUE" business isn't clear. Does CMake have something like an enum? So we could pass, e.g., USE_MOCK_ENGINE instead of TRUE?

Reply:
Can't see how to do it nice with enum (CMake has enums)
Changed the literal TRUE to a variable which explains the purpose, is this better?

Reply:
Oh I got you, you mean to pass to the function something like: USE_MOCK_ENGINE::TRUE
That is not possible, the closest thing to enum in CMake is to check if a variable is within a set of values.
Actually CMake type is always a string, some commands can treat a variable as path / bool but that's it: https://cgold.readthedocs.io/en/latest/tutorials/variables/regular.html#types-of-variable

Reply:
Ok